include/utils/file_utils.php

```diff
@@ -129,6 +129,27 @@ function write_array_to_file($the_name, $the_array, $the_file, $mode="w", $heade
     return sugar_file_put_contents($the_file, $the_string, LOCK_EX) !== false;
 }
 
+function write_override_label_to_file($the_name, $the_array, $the_file, $mode = "w", $header = '')
+{
+    if (!empty($header) && ($mode != 'a' || !file_exists($the_file))) {
```

Comment:
Should use strict comparison.
```suggestion
    if (!empty($header) && ($mode !== 'a' || !file_exists($the_file))) {
```